src/main/java/com/vso/frontEnd/AuthenticationScreen.java

```diff
@@ -0,0 +1,133 @@
+package com.vso.frontEnd;
+
+import com.vso.model.enumaration.LoginStatus;
+import com.vso.model.service.authentication.AuthenticationServiceImpl;
+import com.vso.view.AuthView;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+
+public class AuthenticationScreen extends BaseScreen {
+    AuthView authView;
+    AuthenticationServiceImpl authenticationService;
+    ForgottenPassListener callback1;
+    private final AuthScreenListener callback;
+
+    public AuthenticationScreen(AuthScreenListener callback, ForgottenPassListener callback1) {
+        this.callback1 = callback1;
```

Comment:
It's better if you change the name of the callback and callback1, with something more specific.